engine/src/main/resources/swagger/cromwell.yaml

```diff
@@ -529,6 +529,26 @@ paths:
       security:
         - google_oauth:
             - openid
+  '/workflows/{version}/stats':
```

Comment:
This is actually kinda annoying in swagger IMO but I'm willing to accept that it's like the anger I have for a week after facebook changes its UI